### DISCUSSION TITLE
[Suggestion] '인기 글' 및 '최근 글'에서의 border-radius

### DISCUSSION THREAD

**Discussion by lumirlumir** [pronist/hELLO#249] [Ideas]
안녕하세요, '인기 글' 및 '최근 글'에서의 img에 border-radius를 적용하면 더 좋을 것 같아 이슈를 작성하였습니다.

현재 '인기 글' 및 '최근 글'은 아래와 같이 tailwindCSS 상에서의 border-radius(`rounded-`) 속성이 적용되어 있지 않습니다.

![image](https://github.com/user-attachments/assets/ada2b8d2-0871-4ff8-a50e-d8a878e82a73)

하지만, 사이드바의 '태그'나 '검색 창' 등에는 border-radius가 적용되어 있는 상황이라, 통일성을 위해 '인기 글' 및 '최근 글'의 이미지에 border-radius를 적용하면 좋을 것 같다는 생각입니다.

가능하다면 관련 PR을 올리고 싶습니다.

- 변경 전

![image](https://github.com/user-attachments/assets/66fc74c9-006b-49b9-87ef-11ae2700b91e)

- 변경 후 (검색창(Search)와 동일한 tailwind `rounded-sm` 적용)

![image](https://github.com/user-attachments/assets/1a6b25c0-5e8e-497a-a654-3bab534ebaf8)

**Comment by lumirlumir** [comment]
@pronist 진행을 해도 괜찮은 내용일까요?

**Comment by pronist** [comment]
@lumirlumir 안녕하세요.

`rounded-full` 을 적용한 이미지 이외에 다른 이미지의 경우에는 `border-radius` 를 의도적으로 적용하지 않은 것이므로 이 요청은 받아들이기 어렵습니다.

**Reply by lumirlumir** [reply]
네 알겠습니다 답변 감사드립니다.
